bluesky_queueserver/manager/tests/test_profile_ops.py

```diff
@@ -86,3 +89,49 @@ def test_parse_plan(plan, success, err_msg):
     else:
         with pytest.raises(RuntimeError, match=err_msg):
             parse_plan(plan, allowed_plans=plans, allowed_devices=devices)
+
+
+def test_gen_list_of_plans_and_devices(tmp_path):
+    """
+    Copy simulated profile collection and generate the list of allowed (in this case available)
+    plans and devices based on the profile collection
+    """
+    pc_path = get_default_profile_collection_dir()
+
+    # Copy simulated profile collection (only .py files)
+    file_pattern = os.path.join(pc_path, "[0-9][0-9]*.py")
```

Comment:
I forgot to mention it before, but we may want to exclude `00-base.py` and `01-startup.py` files which use `nslsii.configure_base`, which requires IPython's `get_ipython`, which does not work in pure Python.

Reply:
Tom said that we could patch `get_ipython`, but we didn't return to this discussion. I would suggest to solve this problem separately.